starter-code/app.js

```diff
@@ -26,8 +33,47 @@ app.use(bodyParser.urlencoded({ extended: false }));
 app.use(cookieParser());
 app.use(express.static(path.join(__dirname, 'public')));
 
+app.use(session({
+  secret: "our-passport-local-strategy-app",
+  resave: true,
+  saveUninitialized: true
+}));
+
+//Passpord code
```

Comment:
Better put the passport configuration in its own file (inside a folder like helpers, for example).